src/app/containers/FrontPageMain/index.test.jsx

```diff
@@ -1,11 +1,38 @@
 import React from 'react';
 import FrontPageMain from '.';
 import { shouldShallowMatchSnapshot } from '../../helpers/tests/testHelpers';
-import frontPageDataPidgin from '../../../../data/prod/pidgin/frontpage';
+import frontPageDataIgbo from '../../../../data/prod/igbo/frontpage';
+
+const igboConfig = {
+  brandName: 'BBC News Ìgbò',
```

Comment:
Can we add a comma after BBC News please? e.g. "BBC News, Ìgbò" This applies to all WS brands